update_data.bash

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+set -e
+set -Euo pipefail
+
+# See https://joschi.github.io/java-metadata/ for supported values
+LIST_OS="linux macosx"
+LIST_ARCH="x86_64"
+
+DATA_DIR="./data"
+
+if [[ ! -d "${DATA_DIR}" ]]
+then
+	mkdir "${DATA_DIR}"
+fi
+
+function metadata_url {
+	local os=$1
+	local arch=$2
+
+	echo "https://joschi.github.io/java-metadata/metadata/ga/${os}/${arch}/jdk.json"
```

Comment:
Do we still using `joschi.github.io` ?

Reply:
That's the current source of the JDK metadata: https://joschi.github.io/java-metadata/